Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -190,9 +190,18 @@ If not installing with Helm, here are the steps to install with manifests.
    export KUBECONTEXT=<cluster-name>
    ```
 
-1. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh) script.
+1. Here is the link for [import](https://github.com/rook/rook/blob/master/deploy/examples/external/import-external-cluster.sh) script. The script has used the `rook-ceph-external` namespace and few parameters that also have referenced from namespace variable. If user's external cluster has different namespace, can change parameters in the script according to their external cluster settings. For example with `rook-ceph` namespace, changes needed on the following parameters in the script.
 
-   !!! note
+   ```console
+   NAMESPACE=${NAMESPACE:="rook-ceph"}
+   ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE"-external"
+   CLUSTER_ID_RBD=$NAMESPACE"-external"
+   CLUSTER_ID_CEPHFS=$NAMESPACE"-external"
+   ```
+
+1. Run the import script.
```

Comment:
The numbers don't render correctly when there are too many notes/paragraphs between them, so we need to keep the hard-coded numbering
```suggestion
2. Run the import script.
```